### ISSUE TITLE
Do not display thumbnail placeholder after disabling "Load Thumbnails"

### ISSUE THREAD

**Issue by Jozxyqk6** [TeamNewPipe/NewPipe#8871]
### Checklist

- [X] I made sure that there are *no existing issues* - [open](https://github.com/TeamNewPipe/NewPipe/issues) or [closed](https://github.com/TeamNewPipe/NewPipe/issues?q=is%3Aissue+is%3Aclosed) - which I could contribute my information to.
- [X] I have read the [FAQ](https://newpipe.net/FAQ/) and my problem isn't listed.
- [X] I'm aware that this is a request for NewPipe itself and that requests for adding a new service need to be made at [NewPipeExtractor](https://github.com/TeamNewPipe/NewPipeExtractor/issues).
- [X] I have taken the time to fill in all the required details. I understand that the feature request will be dismissed otherwise.
- [X] This issue contains only one feature request.
- [X] I have read and understood the [contribution guidelines](https://github.com/TeamNewPipe/NewPipe/blob/dev/.github/CONTRIBUTING.md).

### Feature description

Add an option to remove the (placeholder) thumbnail in video lists and place the watch position bar under the entire text.

### Why do you want this feature?

After disabling  "Load Thumbnails" in the Content Settings, the lists of videos still display a placeholder thumbnail, looking rather ugly and taking up space that could be used for larger/more text.

**Comment by SameenAhnaf** [comment]
I agree. We need too big boxes as thumbnails neither. Dots or boxes like 1️⃣,2️⃣,3️⃣, ... on the left of each video title should get the job done.
